browser-test/src/api_docs.test.ts

```diff
@@ -102,6 +108,7 @@ describe('Viewing API docs', () => {
     expect(await page.textContent('html')).toContain(
       'Program and version not found',
     )
+    await page.waitForTimeout(300) // ms
```

Comment:
question (non-blocking): did you add the additional `waitForTimeout` calls because of the issue with inconsistent screenshots commented on below?

Reply:
Yes, specifically because of the open/close caret icon taking a few millis to rotate.